flang/lib/Lower/OpenMP.cpp

```diff
@@ -364,9 +364,12 @@ getScheduleModifiers(const Fortran::parser::OmpScheduleClause &x) {
       const auto &modType2 = std::get<
           std::optional<Fortran::parser::OmpScheduleModifier::Modifier2>>(
           modifier->t);
-      if (modType2->v.v !=
-          Fortran::parser::OmpScheduleModifierType::ModType::Simd)
+      if (modType2 &&
+          modType2->v.v !=
+              Fortran::parser::OmpScheduleModifierType::ModType::Simd)
         return translateModifier(modType2->v);
+      else
```

Comment:
It looks like `else` should be deleted.

Reply:
Assuming this is a coding style comment. Done.
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return